data.sql

```diff
@@ -1,198 +1,11 @@
 /* Populate database with sample data. */
 
```

Comment:
- [ ] The same on the  `species` table ⬇️ Kindly address this. 

![image](https://github.com/elad237/Vet-clinic-database/assets/79058364/aff79344-58ff-492f-83f2-ad12bd725e19)

- Your code may look like this:

  ```sql
   INSERT INTO species (name)
   VALUES ('Pokemon');
  
   INSERT INTO species (name)
   VALUES ('Digimon');
  ```
*